ndk/src/main/jni/utils/stack_unwinder_libunwind.c

```diff
@@ -25,11 +26,21 @@ bsg_libunwind_callback(struct _Unwind_Context *context, void *arg) __asyncsafe {
   bsg_libunwind_state *state = (bsg_libunwind_state *)arg;
 
 #if defined(__arm__)
```

Comment:
I found the intro to the PR quite helpful in understanding how this works. It'd be good to include the explanation as a comment so that future travellers have the same context about the workaround.